explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -499,37 +508,41 @@ private object SpectroscopyModesTable extends TableHooks:
       // By having it as state with the following `useEffectWithDepsBy`, the scrollTo effect will run
       // in the following "hook cycle" and get the proper index.
       .useState(ScrollTo.Scroll)
-      .useEffectWithDepsBy((_, _, _, _, _, _, _, table, _) => table.getState().sorting) {
-        (_, _, _, _, _, _, _, _, scrollTo) => _ => scrollTo.setState(ScrollTo.Scroll)
+      .useEffectWithDepsBy((_, _, _, _, _, _, _, _, table, _) => table.getState().sorting) {
+        (_, _, _, _, _, _, sorting, _, table, scrollTo) => s =>
+          table.setSorting(s) *>
+            sorting.setState(s.value.headOption) *> scrollTo
```

Comment:
I'm not sure the whole mechanism will work if time is not the first column in the sort. Eg: we can sort first by instrument and then by time.

Reply:
II don't think multi sort is enabled in this table. AFAIK only one column is sorted